Airports/EGKK/SMR/Regions.txt

```diff
@@ -2124,12 +2146,11 @@ N051.09.20.362 W000.10.20.454
 N051.09.20.331 W000.10.20.545
 N051.09.20.262 W000.10.20.544
 N051.09.20.160 W000.10.20.530
-N051.09.15.715 W000.10.19.093
-N051.09.15.572 W000.10.18.966
-N051.09.15.492 W000.10.18.891
-N051.09.15.446 W000.10.18.713
-N051.09.15.361 W000.10.18.463
-N051.09.15.428 W000.10.17.897
+N051.09.17.780 W000.10.19.696
+N051.09.18.230 W000.10.16.661
+N051.09.17.913 W000.10.16.530
+N051.09.17.913 W000.10.16.638
+N051.09.15.721 W000.10.15.984
```

Comment:
what do these changes relate to?

Reply:
The whole pier (stands 27-10, etc) is made in one region, so I just removed stand 27 for it so the grey wouldn't show up

Reply:
We might want that again when stand 27 is returned to service, so either, comment out the relevant parts and retain them for later, or draw the smrRed section "later" in the process so it draws on top

Reply:
Right, I'll just draw on top of it then